runtime/themes/iceberg-light.toml

```
@@ -0,0 +1,39 @@
+# Author : Chromo-residuum-opec <development.0extl@simplelogin.com>
+
+inherits = "iceberg-dark"
+ui.menu.selected = { fg = "background_fg", bg = "menusel_bg" }
```

```suggestion
"ui.menu.selected" = { fg = "background_fg", bg = "menusel_bg" }
```

This needs to be quoted so the entire `ui.menu.selected` string is used as a key (rather than a nested TOML table)

---

updated in https://github.com/helix-editor/helix/pull/10674/commits/9e49006c4039bba9ad0c2d9ff4168e1c0169b3f6